index.html

```diff
@@ -0,0 +1,550 @@
+          </div>
+          <p>Sarah Drasner & Geoff Graham <span>wrote</span></p>
+          <h3>How To Create A Headless WordPress Site On The JAMstack</h3>
+          <span><i class="fab fa-comment-alt fa-2x fa-fw"></i> 14 comments</span>
+          <div class="first-div-floats">
+            <p># Wordpress</p>
+            <p># Static Generators</p>
+          </div>
+        </article>
+      </div>
+    </section>
+    <!-- latest post -->
+    <section>
+      <h2>latest post</h2>
+      <div class="latest-post bg-gray-1">
+        <div class="md">
+          <p>3 DAYS AGO</p>
+
+          <h3>
+            An Interview With Zach Leatherman: A SmashingConf Austin Speaker
+          </h3>
+
+          <p>By Rachel Andrew</p>
+
+          <p>
+            SmashingConf is coming to Austin, and in this post, we bring you
+            an interview with one of our speakers, Zach Leatherman, who will
+            be talking about type and font performance.
+          </p>
+        </div>
+
+        <div class="md">
+          <p>6 DAYS AGO</p>
+
+          <h3>29 Days Of February (2020 Wallpapers Edition)</h3>
+
+          <p>By Cosima Mielke</p>
+
+          <p>
+            Artists and designers from across the globe once again got out
+            their favorite tools and created unique and inspiring wallpapers
+            to welcome the new month. The wallpapers are available with...
+          </p>
+        </div>
+
+        <div class="md">
+          <p>8 DAYS AGO</p>
+
+          <h3>
+            Using CSCS Scripting Language For Cross-Platform Development
+          </h3>
+
+          <p>By Vassili Kaplan</p>
+
+          <p>
+            In this article, Vassili Kaplan explains how to use a scripting
+            language to develop cross-platform mobile applications.
+          </p>
+        </div>
+
+        <div class="md">
+          <p>our friends and supporters</p>
+        </div>
+
+        <div class="lg">
+          <div class="lg-img"></div>
+
+          <h3>1,651 smashing members</h3>
+
+          <p>
+            A friendly community that keeps this website alive. With books,
+            webinars, discounts and networking — for just 1 coffee a month.
+            Join the family.
+          </p>
+
+          <button>Explore features →</button>
+
+          <p>Already have an account? Sign in!</p>
+        </div>
+
+        ​
+
+        <div class="md">
+          <p>9 DAYS AGO</p>
+
+          <h3>
+            How Smashing Magazine Manages Content: Migration From WordPress To
+            JAMstack
+          </h3>
+
+          <p>By Sarah Drasner</p>
+
+          <p>
+            WordPress adoption is massive. So why would a WordPress site
+            consider moving to JAMstack? In this technical case study, we’ll
+            cover what an actual WordPress migration looks like, using
+            Smashing Magazine itself! We’ll talk through the gains and ….
+          </p>
+        </div>
+
+        <div class="md">
```

Comment:
line #180 should be `<article>` and not a `<div>`

Reply:
done